Issue with current CLI version used by the action

Hello,

With the current cli version on the action (`@hubspot/cms-cli@2.2.0`) I see the following issue:

```
Cannot find module '@hubspot/cms-lib/api/function'
Require stack:
- /home/runner/.npm/_npx/1733/lib/node_modules/@hubspot/cms-cli/commands/logs.js
- /home/runner/.npm/_npx/1733/lib/node_modules/@hubspot/cms-cli/bin/cli.js
- /home/runner/.npm/_npx/1733/lib/node_modules/@hubspot/cms-cli/bin/hs
 ```

The version of the CLI I have on the `package.json` is: 
```
"@hubspot/cms-cli": "^2.2.0"
```

I did manually run the same action with bumped patch version `@hubspot/cms-cli@2.2.1` and it worked fine. Can we maybe get this updated? I'm happy to do a PR if it's needed :)

Thank you very much!

--- a/action.yml
+++ b/action.yml
@@ -28,5 +28,5 @@ runs:
         HUBSPOT_PORTAL_ID: ${{ inputs.portal_id }}
         HUBSPOT_PERSONAL_ACCESS_KEY: ${{ inputs.personal_access_key }}
       run: |
-        npx @hubspot/cms-cli@2.2.1 upload ${{ inputs.src_dir }} ${{ inputs.dest_dir }} --use-env
+        npx @hubspot/cms-cli@2.2.2 upload ${{ inputs.src_dir }} ${{ inputs.dest_dir }} --use-env
       shell: bash

